@noun-auction/components/DataRenderers/AuctionHighBid.tsx

```diff
@@ -34,16 +34,16 @@ export function AuctionHighBid({
           color={layout === 'sideBarBid' ? 'tertiary' : 'secondary'}
           mb={layout === 'sideBarBid' ? 'x2' : 'x0'}
         >
-          {label}
+          {timerComplete ? 'Winning bid' : label}
         </Label>
       )}
-      {auctionData?.rpcData?.amount && (
+      {activeAuction?.properties?.highestBidPrice?.chainTokenPrice?.raw && (
```

Comment:
For clarity, I'd suggest plugging this into a more readable variable name at the top of the func

Reply:
so like this then? and this does not need to be memoized, or should we with useMemo.

```
const { layout, timerComplete, activeAuction } = useNounishAuctionProvider()

const highestBid = activeAuction?.properties?.highestBidPrice?.chainTokenPrice?.raw

return (...

   {highestBid && (
      <EthAmount
        style={{ lineHeight: '1.15' }}
        size="md"
        align={layout === 'sideBarBid' ? 'left' : 'right'}
        className={layout === 'sideBarBid' && sidebarHighBid}
        ethAmount={highestBid}
      />
    )}
```

Reply:
I'm in the habit of using it, but that's just a reflex at this point. In practice I think it's only really necessary when the function is doing a bit of lifting, because the function call is often just as heavy as the resolve.